src/test/scala/edu/ie3/simona/config/ConfigFailFastSpec.scala

```diff
@@ -39,18 +39,46 @@ class ConfigFailFastSpec extends UnitSpec with ConfigTestData {
           }
         }
 
-        "identify invalid input" in {
+        "identify invalid date or time configuration" in {
           intercept[InvalidConfigParameterException] {
-            ConfigFailFast invokePrivate checkDateTime(
+            ConfigFailFast invokePrivate checkTimeConfig(
               new Time(
                 "2020-06-18 13:41:00",
                 None,
-                "total non-sense",
+                "2020-07-18 13:41:00",
                 true
               )
             )
-          }.getMessage shouldBe "Invalid dateTimeString for config parameter simonaConfig.simona.time.startDateTime: total non-sense. Please " +
-            "ensure that your date/time parameter match the following pattern: ‘yyyy-MM-dd HH:mm:ss'"
+          }.getMessage shouldBe "Invalid time configuration." +
+            "Please ensure that the start time of the simulation is before the end time."
+        }
+      }
+
+      "Checking date string" should {
+        val createDateTime =
+          PrivateMethod[ZonedDateTime](Symbol("createDateTime"))
+
+        val dateTimeString: String = "2020-05-18 13:41:00"
+
+        "let valid input pass" in {
+          noException shouldBe thrownBy {
+            ConfigFailFast invokePrivate createDateTime(
+              (
```

Comment:
There's some obsolete parenthesis here. Doesn't really hurt, but it'd probably be easier to read the code if they were not there. Same below.